centreon/www/front_src/src/ResourceAccessManagement/Filter/index.tsx

```diff
@@ -0,0 +1,39 @@
+import { useRef } from 'react';
+
+import { useTranslation } from 'react-i18next';
+import { useSetAtom } from 'jotai';
+
+import debounce from '@mui/utils/debounce';
+
+import { SearchField } from '@centreon/ui';
+
+import { searchAtom } from '../atom';
+import { labelSearch } from '../translatedLabels';
+
+const Filter = (): JSX.Element => {
+  const { t } = useTranslation();
+
+  const setSearchValue = useSetAtom(searchAtom);
+
+  const searchDebounced = useRef(
+    debounce<(search) => void>((debouncedSearch): void => {
+      setSearchValue(debouncedSearch);
+    }, 500)
+  );
```

Comment:
It's not mandatory to use a ref in this case
```suggestion
  const searchDebounced = debounce<(search) => void>((debouncedSearch): void => {
      setSearchValue(debouncedSearch);
    }, 500);
```